src/handlers/emojis/GUILD_EMOJIS_UPDATE.ts

```diff
@@ -11,7 +11,7 @@ export async function handleGuildEmojisUpdate(data: DiscordGatewayPayload) {
 
   const cachedEmojis = guild.emojis;
   guild.emojis = new Collection(
-    payload.emojis.map((emoji) => [emoji.id ?? emoji.name, emoji]),
+    payload.emojis.map((emoji) => [emoji.id!, emoji]),
```

Comment:
Explain why this is done

Reply:
This event is only emitted when a custom emoji is updated which always has an id.